crud/src/main/java/com/redhat/lightblue/rest/crud/cmd/RunSavedSearchCommand.java

```diff
@@ -70,15 +66,19 @@ public RunSavedSearchCommand(String searchName,
         this.from=from;
         this.to=to;
         this.params=properties;
+        this.metrics = metrics;
     }
 
     @Override
     public CallStatus run() {
+        RequestMetrics.Context savedSearchMetricCtx = metrics.startEntityRequest("savedsearch", entity, version);
```

Comment:
Can we use the saved search name in the metrics for this actually? That will tell us exactly how certain queries are performing. Because this has an additional parameter, I'd probably track this as `startSavedSearchRequest`.

Reply:
Added startSavedSearchRequest to track monitoring of savedSearch with searchName as an added parameter.